scaffolds_actor_model/python/src/my_module.py

```diff
@@ -0,0 +1,28 @@
+import thespian.actors as ta
+
+class MyClass(ta.Actor):
+     def __init__(self, **kwargs):
+          super().__init__(**kwargs)
+          self.neighbour = None
+
+     def receiveMessage(self, msg, sender):
+          if msg["msg_type"] == "receive_address":
+               self.neighbour = msg["address"]
```

Comment:
The use of *neighbour* makes me think that you had a particular use case (our experiment with a concurrent GoL), but this example should be use-case agnostic. Maybe renaming the variable is enough ?

WDYT ?